Redact additional keys in logs of Flows

paescuj opened the pull request:
Closes #18353 / ENG-931

paescuj committed:
Redact additional keys in logs of Flows

paescuj committed:
Create moody-poems-pump.md

paescuj committed:
Move REDACTED_TEXT to constants package & update tests

paescuj committed:
Revert "Move REDACTED_TEXT to constants package & update tests"

This reverts commit 0f5b22725380b858fbccc1559760ba9fb0e4ac5d.

paescuj committed:
Update redacted value in blackbox test

rijkvanzanten commented on api/package.json:
```diff
@@ -162,7 +161,9 @@
 		"vm2": "3.9.17",
-		"wellknown": "0.5.0"
+		"wellknown": "0.5.0",
+		"wild-wild-parser": "^3.4.0",
+		"wild-wild-path": "^3.6.0"
```
Why do we need this? How are they different from `lodash`'s `get`/`set`?

rijkvanzanten commented on api/src/flows.ts:
```diff
@@ -8,37 +9,36 @@ import type {
 import { omit, pick } from 'lodash-es';
 import { get } from 'micromustache';
 import { schedule, validate } from 'node-cron';
+import { REDACTED_TEXT } from './constants.js';
```
Lets put this in `@directus/constants` 👍🏻

paescuj committed:
Merge branch 'main' into redact-additional-keys-flow-logs

paescuj committed:
Use own redact implementation

paescuj committed:
Move REDACTED_TEXT to constants package

paescuj committed:
Replace outdated comment

paescuj committed:
Fix misleading return type

Since values might change (redacted), output isn't necessarily the same type anymore
